### PR TITLE
Resources Submit Page

### PR DESCRIPTION
**Pull request by mazma1**
For [Issue 16](https://github.com/codebuddies/react-concept/issues/16)

#### What was done:
- Created a form for submitting resources.
- Included necessary event handlers

#### Preview:

Screenshot 2019-10-25 at 9 11 58 PM

### CODE REVIEW

File: src/components/Resources/submitResource.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    <Grid container spacing={1}>
+        <Grid item lg={3}>
+          <PersonalMenu />
+        </Grid>
+
+        <Grid item lg={9}>
+          <div className={clsx(classes.root, classes.breadcrumb)}>
+            <Paper elevation={0} className={classes.paper}>
+              <Breadcrumbs separator={<NavigateNextIcon fontSize="small" />} aria-label="breadcrumb">
+                <Link color="inherit" href="#">
```
Comment (lpatmo) [inline src/components/Resources/submitResource.js]: Could we have this link to `<Link to="/resources">`?

File: src/components/Resources/submitResource.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                name: 'media-type',
+                id: 'outlined-media-type-native-simple',
+              }}
+              className={classes.dense}
+              margin="dense"
+              labelWidth={labelWidth}
+              onChange={handleChange('mediaType')}
+            >
+              <option value="" />
+              <option value='pdf'>PDF</option>
```
Comment (lpatmo) [inline src/components/Resources/submitResource.js]: Could we add the following here too?

<option value='podcast'>Podcast</option>
<option value='tutorial'>Tutorial</option>
<option value='course'>Course</option>
<option value='book'>Book</option>
<option value='game'>Game</option>
<option value='blog'>Blog post</option>
<option value='event'>Event</option>
<option value='website'>Website</option>

**Comment by mazma1** [comment]
@lpatmo thanks for the review, I've made the suggested changes. 😀

**Comment by lpatmo** [comment]
@mazma1 Thank you! :)

**Pull request merged**